Resources/Prototypes/DeltaV/radio_channels.yml

```
@@ -1,6 +1,13 @@
+- type: radioChannel
+  id: Justice
+  name: chat-radio-justice
+  keycode: "j"
+  frequency: 1420
+  color: "#701442"
+
 - type: radioChannel
   id: Prison
   name: chat-radio-prison
   keycode: 'p'
   frequency: 1601
-  color: "#FFA500"
\ No newline at end of file
+  color: "#FFA500"
```

```suggestion
  color: "#FFA500"

```
Try fix whitespace pls